src/Components/TMS/Organisms/FOTMSDashboardNavbar/index.tsx

```diff
@@ -1,20 +1,17 @@
 import React from "react";
```

Comment:
All files inside `components/TMS` must be deleted because we will not use them again.